templates/views/partials/field-tel.twig

```diff
@@ -0,0 +1,12 @@
+<div class="p4fb__field p4fb__field-{{ field.type | e('html_attr') }} p4fb__field-{{ field.name | e('html_attr') }}">
+    <div class="p4fb__field__element p4fb__field__element--{{ field.type | e('html_attr') }} form-group">
+        <input class="{{ field.class }} p4fb__field__input p4fb__field__input--{{ field.type | e('html_attr') }} form-control"
+               type="tel"
+               pattern="\d{2}-(\d{8}|\d{9})"
```

Comment:
Is this going to work for non-US numbers?

Reply:
this is the format according to the specs they provided https://www.greenpeace.org.br/human-made-crm-integration